api_docs/unmerged.d/ZF-85f248

```diff
@@ -0,0 +1,4 @@
+* [`POST /messages/{message_id}/typing`](/api/set-typing-status-for-message-edit):
+  Added a new endpoint `POST /messages/{message_id}/typing` which deprecates the 
+  `POST /message_edit_typing` in order to include `message_id` in the URL path,
+  instead of passing it as a request body parameter.
```

Comment:
This doesn't matter today, but generally you should make sure your editor adds newlines at end-of-file by default.